apps/vr-tests/src/utilities/StoryWrightDecorator.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import type { DecoratorFn } from '@storybook/react';
+import { StoryWright, Step } from 'storywright';
+
+/**
+ * A decorator that wraps the story in a `StoryWright` component with the provided steps.
+ * Also allows to pass steps as a parameter to the story.
+ *
+ * @param steps - The steps to pass to the `StoryWright` component.
+ * @returns The decorator function.
+ *
+ * @example
+ * ```tsx
+ * import { Steps } from 'storywright';
+ *
+ * export default {
+ *  title: 'ExampleComponent',
+ *  decorators: [withStoryWrightSteps(steps)],
```

Comment:
the implementation exports token `StoryWrightDecorator` while example uses `withStoryWrightSteps`, lets update one or the another